pkg/render/manager_test.go

```diff
@@ -79,7 +86,6 @@ var _ = Describe("Tigera Secure Manager rendering tests", func() {
 			{name: networkpolicy.TigeraComponentDefaultDenyPolicyName, ns: "tigera-manager", group: "projectcalico.org", version: "v3", kind: "NetworkPolicy"},
 			{name: render.ManagerServiceAccount, ns: render.ManagerNamespace, group: "", version: "v1", kind: "ServiceAccount"},
 			{name: render.ManagerClusterRole, ns: "", group: "rbac.authorization.k8s.io", version: "v1", kind: "ClusterRole"},
-			{name: render.ManagerClusterRoleBinding, ns: "", group: "rbac.authorization.k8s.io", version: "v1", kind: "ClusterRoleBinding"},
```

Comment:
The cluster role binding moved to a new component - I think we need to add test cases for that new component as well to maintain coverage. 

Basically this: https://github.com/tigera/operator/pull/2715/files#diff-a95b82b76390b16587aaf2a6d12d9ff332d5597002643a3846b518a914aebce0R96-R128